understanding/sensory-characteristics.html

```diff
@@ -65,7 +64,8 @@ <h2>事例</h2>
                <li><p><strong>事例 2: 複数のページによるオンライン調査</strong></p>
                   
                   
-                  <p>複数のページによるオンライン調査では、あるページから次のページへ遷移するためのリンクに緑色の矢印のアイコンを使っていて、それをコンテンツの右下に置いている。矢印には「次へ」というラベルがはっきりと記述されていて、「次の調査へ進むには、最後の質問の右下にある『次へ』というラベルの付いた緑色の矢印のアイコンを選択してください。」という説明文が書かれている。この例では、アイコンを特定できるように、配置、色及びラベルを用いている。 </p>
+                  <p>複数のページによるオンライン調査では、あるページから次のページへ遷移するためのリンクに緑色の矢印のアイコンを使っていて、それをコンテンツの右下に置いている。矢印には「次へ」というラベルがはっきりと記述されていて、「次の調査へ進むには、最後の質問の右下にある『次へ』というラベルの付いた緑色の矢印のアイコンを選択してください。」という説明文が書かれている。この説明文は、アイコンの識別を助けるために位置及び色を用いているが、こうした感覚的な特徴のみに依存するのではなく、ラベルも参照しているため、この達成基準を満たしている。
```

Comment:
> uses a link implemented as a green arrow icon placed

「緑色の矢印のアイコンとして実装されたリンクを使っていて、」ですかね

```suggestion
                  <p>複数のページによるオンライン調査では、あるページから次のページへ遷移するための緑色の矢印のアイコンとして実装されたリンクを使っていて、それをコンテンツの右下に置いている。矢印には「次へ」というラベルがはっきりと記述されていて、「次の調査へ進むには、最後の質問の右下にある『次へ』というラベルの付いた緑色の矢印のアイコンを選択してください。」という説明文が書かれている。この説明文は、アイコンの識別を助けるために位置及び色を用いているが、こうした感覚的な特徴のみに依存するのではなく、ラベルも参照しているため、この達成基準を満たしている。
```

Reply:
そうですね。その線で、修正します。